osu.Game/osu.Game.csproj

```diff
@@ -18,7 +18,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="2.1.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite.Core" Version="2.1.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <PackageReference Include="ppy.osu.Framework" Version="2018.619.0" />
```

Comment:
This is just here for testing purposes I assume? It will be reverted before merge?

Reply:
See depending framework change above

Reply:
If you add the [appveyor nuget source](https://github.com/ppy/osu-framework/wiki/Testing-framework-PRs-with-other-projects-via-NuGet) you can test this change as-is.